### PR TITLE
[ART-4315] rhcos: release-browser moved to new cluster

### PR DESCRIPTION
**Pull request by sosiouxme**
Pulls in
* https://github.com/openshift/doozer/pull/635
* https://github.com/openshift/elliott/pull/400

### CODE REVIEW

**Comment by sosiouxme** [comment]
/hold
this will need adjustment after those merge.

**Review by vfreex — APPROVED** [review]
/lgtm

**Pull request merged**
